site/docs/releases.md

```diff
@@ -64,28 +64,36 @@ High-level features:
   * Added a new, more flexible compaction action for Spark that can support different strategies such as bin packing and sorting. [[\#2501](https://github.com/apache/iceberg/pull/2501), [\#2609](https://github.com/apache/iceberg/pull/2609)].
   * Added the ability to upgrade to v2 or create a v2 table using the table property format-version=2  [[\#2887](https://github.com/apache/iceberg/pull/2887)].
+* **Flink**
+  * Added support for SQL primary keys [[\#2410](https://github.com/apache/iceberg/pull/2410)].
+* **Hive**
+  * Added the ability to set the catalog at the table level in the Hive Metastore. This makes it possible to write queries that reference tables from multiple catalogs [[\#2129](https://github.com/apache/iceberg/pull/2129)].
+  * Deprecated the configuration property `iceberg.mr.catalog` which was previously used to configure the Iceberg catalog in MapReduce and Hive [[\#2565](https://github.com/apache/iceberg/pull/2565)].
+  * Enabled dropping HMS tables despite metadata problems [[\#2583](https://github.com/apache/iceberg/pull/2583)].
+  * Added table-level JVM lock on commits[[\#2547](https://github.com/apache/iceberg/pull/2547)].
+  * Use Hive's vectorized ORC reader [[\#2613](https://github.com/apache/iceberg/pull/2613)].
 * **Spark**
-  * [[\#2560](https://github.com/apache/iceberg/pull/2560)] added extensions DDL to set identifier fields.
-  * [[\#2365](https://github.com/apache/iceberg/pull/2365)] added support for `ALTER TABLE table CHANGE PARTITION FIELD transform TO transform` DDL.
-  * [[\#2660](https://github.com/apache/iceberg/pull/2660)] added support for micro-batch streaming reads for structured streaming in Spark3.
-  * [[\#2777](https://github.com/apache/iceberg/pull/2777)] improves the performance of importing a Hive table by not loading all partitions from Hive and instead pushing the partition filter to the Metastore.
-  * [[\#2193](https://github.com/apache/iceberg/pull/2193), [\#2206](https://github.com/apache/iceberg/pull/2206)] added support for UPDATE statements in Spark.
-* **Flink** now support SQL primary keys [[\#2410](https://github.com/apache/iceberg/pull/2410)].
+  * Added extensions DDL to set identifier fields [[\#2560](https://github.com/apache/iceberg/pull/2560)].
+  * Added support for `ALTER TABLE REPLACE PARTITION FIELD` DDL [[\#2365](https://github.com/apache/iceberg/pull/2365)].
+  * Added support for micro-batch streaming reads for structured streaming in Spark3 [[\#2660](https://github.com/apache/iceberg/pull/2660)].
+  * Improved the performance of importing a Hive table by not loading all partitions from Hive and instead pushing the partition filter to the Metastore [[\#2777](https://github.com/apache/iceberg/pull/2777)].
+  * Added support for UPDATE statements in Spark [[\#2193](https://github.com/apache/iceberg/pull/2193), [\#2206](https://github.com/apache/iceberg/pull/2206)].
```

Comment:
A few more for Spark:
* Spark 3.1 is now supported (#2512)
* Added RemoveReachableFiles action (#2415)
* Added add_files stored procedure (#2210)
* Refactored Actions API and added a new entry point (need to find the ref)
* Added support for Hadoop configuration overrides (#2922)

Reply:
Fixed.